test/e2e/specs/editor/blocks/navigation-frontend-interactivity.spec.js

```diff
@@ -65,67 +65,22 @@ test.describe( 'Navigation block - Frontend interactivity', () => {
 			// Test: overlay menu focuses on first element after opening
 			await expect( overlayMenuFirstElement ).toBeFocused();
 
-			// Test: overlay menu traps focus
-			await pageUtils.pressKeys( 'Tab', { times: 2, delay: 50 } );
-			await expect( closeMenuButton ).toBeFocused();
-			await pageUtils.pressKeys( 'Shift+Tab', { times: 2, delay: 50 } );
-			await expect( overlayMenuFirstElement ).toBeFocused();
-
 			// Test: overlay menu closes on click on close menu button
 			await closeMenuButton.click();
 			await expect( overlayMenuFirstElement ).toBeHidden();
 
 			// Test: overlay menu closes on ESC key
-			await openMenuButton.click();
-			await expect( overlayMenuFirstElement ).toBeVisible();
-			await pageUtils.pressKeys( 'Escape' );
-			await expect( overlayMenuFirstElement ).toBeHidden();
-			await expect( openMenuButton ).toBeFocused();
-		} );
-
-		/**
-		 * These are already tested within the Overlay Interactions test above, but Safari is flakey on the Tab
-		 * keypresses (passes 50 - 70% of the time). Tab keypresses are testing fine manually in Safari, but not
-		 * in the test. Once we figure out why the Tab keypresses are flakey in the test, we can
-		 * remove this test and only rely on the Overlay Interactions test above and add a (@firefox, @webkit)
-		 * directive to the describe() statement. https://github.com/WordPress/gutenberg/pull/55198
-		 */
-		test( 'Overlay menu interactions in Safari (@webkit)', async ( {
```

Comment:
Does this test no longer work or apply?

Reply:
This test duplicates the test above it. As the comment preceding it indicates, the intention was to avoid flakiness of the test in Safari purportedly caused by Tab key presses. I removed those parts of the test so I followed up with the rest of what the comment recommends.

The reason I removed the assertions for Tab key and focus is because the “focus trapping“ implementation is removed and testing for it is not needed. The “focus trapping” can be removed because the `dialog` element, as a modal, makes all content outside it inert.

Aside: this test was a complete duplicate of the one above it and included Tab key presses so I don’t see how it was actually making any difference for Safari in the first place.